### DISCUSSION TITLE
Migrate Firefly 3 from RaspberryOS to Ubuntu

### DISCUSSION THREAD

**Discussion by kikinhocd** [firefly-iii/firefly-iii#4802] [General]
I am running Firefly III on RaspbianOS 64-bit and now I want to start using Ubuntu 64-bit on my Rasberry Pi 4b.

I use Firefly 3 in a docker-compose and I want to continue using it on my new setup.

What should I need to do to migrate the database?

Please detailed explanation, since I am not a programmer and I did the installation using step-by-step guide on YouTube.

**Comment by JC5** [comment]
I appreciate that this can be difficult, but you have to realize nobody is just going to write this huge detailed guide for you just because you ask.

**Reply by kikinhocd** [reply]
I don't ask for step-by-step guide, I ask for detailed explanation.

I have read about a procedure where a guy was migrating on devices from same OS, but newer version and he made backup of his MySQL database and things worked for him only after he first  installed and started MySQL with the backup of the old database and only after that he installed Firefly 3 with same docker-compose.yml and .env files. But will this type of migration work for different OS and also I don't know how to do backup of my MariaDB:)

**Reply by JC5** [reply]
All I'm saying is, if you have a detailed question you can get a detailed explanation.

**Comment by niklas2810** [comment]
Which database system do you have installed as a container? (e.g. MySQL, Postgres)

**Reply by kikinhocd** [reply]
I use MariaDB
